packages/manager/modules/pci/src/projects/project/private-registry/onboarding/onboarding.html

```diff
@@ -0,0 +1,17 @@
+<pci-project-empty data-guides="$ctrl.guides" data-image-source="$ctrl.privateRegistryImage">
+    <h1 data-translate="private_registry_title"></h1>
+    <p data-translate="private_registry_onboarding_content1"></p>
+    <strong data-translate="private_registry_onboarding_content2"></strong>
+    <p data-translate="private_registry_onboarding_content3"></p>
+
+    <cui-message-container data-messages="$ctrl.messages" class="mb-2 mt-2"></cui-message-container>
+    <oui-button data-variant="primary"
+                data-on-click="$ctrl.createRegistry()" data-ng-if="!$ctrl.registryId">
```

Comment:
Does it make sense to have a link with a `ui-sref` instead ?

Reply:
It wont look good as a link as we are displaying it in a new line below a message.

Reply:
Does applying the `oui-button` class solves it ?